src/types/permission-types.ts

```diff
@@ -73,6 +73,7 @@ export type PermissionScope = ProtocolPermissionScope | MessagesPermissionScope
 export type ProtocolPermissionScope = {
   interface: DwnInterfaceName.Protocols;
   method: DwnMethodName.Configure | DwnMethodName.Query;
+  protocol?: string;
```

Comment:
Wondering if it's really fine allowing configuration of any protocol, essentially giving god power. 🤔

Reply:
I was thinking about this... I'm good with restricting it! I just wasn't sure since it was optional beforehand already giving god power.

Wonder if there are any use-cases for god power w/ protocols that we want vs giving up the whole private key?

Reply:
Okay for us to leave it as is as long as we have discussed this and on the same page on understanding.